pallets/collator-selection/src/lib.rs

```diff
@@ -292,6 +292,17 @@ pub mod pallet {
 					"invulnerables > T::MaxInvulnerables; you might need to run benchmarks again"
 				);
 			}
+
+			// check if the invulnerables have associated validator keys before they are set
+			for account_id in new {
```

Comment:
CI is failing due to ownership issues on this vec, use a reference to avoid:

```suggestion
			for account_id in &new {
```